app/controllers/events_controller.rb

```diff
@@ -0,0 +1,53 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[show edit update attend]
+
+  # GET /events or /events.json
+  def index
+    @events = Event.all
+  end
+
```

Comment:
- [ ] Events index action should list past events, i.e. `@past_events = Event.previous_events`
- [ ] Events index action should list upcoming events, i.e. `@upcoming_events = Event.upcoming_events`